var/spack/repos/builtin/packages/py-niworkflows/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyNiworkflows(PythonPackage):
+    """Common workflows for MRI (anatomical, functional, diffusion, etc)"""
+
+    homepage = "https://github.com/nipreps/niworkflows"
+    pypi     = "niworkflows/niworkflows-1.4.0.tar.gz"
+
+    version('1.4.0', sha256='d4e59070fde0290e0bfeece120ff1d2ff1f9573e3f2e6a719fe463c913af25ec')
+    version('1.3.5', sha256='92e24f3462fb3ad4d8ee724506fba05da2b3ca0626850dd2e637a553e17d69b8')
+    version('1.0.4', sha256='34bfa5561e6f872dbd85bb30a1b44c5e1be525167abe3932aee8ac06d15f6ed9')
+
+    variant('fsl', default=False, description="Enable fsl support.")
+    variant('ants', default=False, description="Enable ants support.")
+
+    depends_on('python@3.7:', when='@1.3.3:', type=('build', 'run'))
+    depends_on('python@3.6:', when='@1.2:', type=('build', 'run'))
+    depends_on('python@3.5:', type=('build', 'run'))
+    # versions from setup.py and pyproject.toml differ
+    # -> take the one from setup.py
+    depends_on('py-setuptools@40.8.0:', when='@1.2.3:', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@38.4.1:', when='@1.2.3:', type='build')
```
We don't run `python setup.py sdist`.